mazerunner/mouse.cpp

```diff
@@ -228,7 +228,7 @@ void Mouse::turn_SS90EL() {
   while (not forward.is_finished()) {
     delay(2);
   }
-  forward.set_position(170);
+  forward.set_position(FULL_CELL - 10);
```

Comment:
Same as above, 10.0 vs. 10

Reply:
I think these don't matter practically. I have nevertheless updated them and fixed a couple of other minor issues. 

Probably, these should be explicitly float rather than implied double as well but i will leave it for now.